### DISCUSSION TITLE
Update dependencies without version specified explicitly?

### DISCUSSION THREAD

**Discussion by ezloj** [renovatebot/renovate#13098] [Request Help] [answered]
Is it possible to update dependencies without version specified explicitly?

For example, a dependency such as:
```
<dependency>
            <groupId>net.sf.opencsv</groupId>
            <artifactId>opencsv</artifactId>
</dependency>
```

Would get changed into:
```
<dependency>
            <groupId>net.sf.opencsv</groupId>
            <artifactId>opencsv</artifactId>
            <version>2.3</version>
</dependency>
```
where 2.3 is just a latest available version for that dependency at the time of running the renovate bot

**Comment by rarkins** [comment, marked as answer]
No, this is not currently possible. We would consider this to be an advanced form of "pinning".

**Reply by ezloj** [reply]
thank you for quick reply!
